dev/Common/CornerRadiusToFilteredMultipliedThicknessConverter.cpp

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <pch.h>
+#include <common.h>
+#include "CornerRadiusToFilteredMultipliedThicknessConverter.h"
+
+winrt::Thickness CornerRadiusToFilteredMultipliedThicknessConverter::Convert(
+    winrt::CornerRadius const& radius,
+    winrt::CornerRadiusToThicknessSide const& side,
+    double multiplier)
+{
+    auto result = winrt::Thickness{};
+
+    switch (side)
+    {
+    case winrt::CornerRadiusToThicknessSide::Left:
+        result.Left = radius.BottomLeft * multiplier;
+        result.Right = 0;
+        result.Top = 0;
+        result.Bottom = 0;
+        break;
+    case winrt::CornerRadiusToThicknessSide::Top:
+        result.Left = 0;
+        result.Top = radius.TopLeft * multiplier;
```

Comment:
>radius.TopLeft * multiplie [](start = 21, length = 26)

How did you decide that this should come from radius.TopLeft instead of radius.TopRight?

Reply:
I "rotated" the corner radii by 45° clockwise. That way, bottomleft becomes left, topleft becomes top etc. The 45° clockwise were chosen arbitrarily.

Reply:
Should we do something like take for result.Top take the max of TopLeft and TopRight?

Reply:
Given that this converter is removed now, should we add this behavior to the CornerRadiusToThicknessConverter? With the current state of the PR, developers would be able to choose whether the left thickness should come from TopLeft or BottomLeft of the corner radius.